pkg/trace/sampler/scoresampler.go

```diff
@@ -20,9 +20,6 @@ const (
 	shrinkCardinality = 200
 )
 
-// ErrorsSampler is dedicated to catching traces containing spans with errors.
```

Comment:
Moved Errors sampler to a dedicated file since I added quite a few methods. This is consistent with the priority sampler and the rare sampler having their own files :)